tests/mrcc/MINP

```diff
@@ -0,0 +1,23 @@
+# Local CCSD(T) for a five H2 cluster with cc-pVDZ basis testing the 2018 scheme with vloose settings
+mem=50mb
+calc=LNO-CCSD(T)
+basis=cc-pVDZ
+
+localcc=2018
+lcorthr=vloose
+
+test=-5.662212025812
```

Comment:
Does the run fail if the single point calculation does not give exactly this value?

Reply:
I guess it would. But I actually had never any failing test in the MRCC test suite.